ldk/javascript/examples/self-test-loop/package.json

```diff
@@ -44,7 +44,14 @@
       "filesystem": {},
       "process": {},
       "ui": {},
-      "user": {},
+      "user": {
+        "optionalClaims":[
+          {
+            "value" :"add your email address here"
```

Comment:
Should this include an actual email address for the self test loop?

Reply:
I used my email on my end 0.0. Do you volunteer to put your email here? Or I can just put Brett's 0,0

Reply:
@MaggieMaSS if we use a Google Group email for our team, that should do the trick!